docs/data-sources/group.md

```diff
@@ -0,0 +1,32 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_group Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Groups are how users are authorized to access Resources. For more information, see Twingate's documentation https://docs.twingate.com/docs/groups.
+---
+
+# twingate_group (Data Source)
+
+Groups are how users are authorized to access Resources. For more information, see Twingate's [documentation](https://docs.twingate.com/docs/groups).
+
+## Example Usage
+
+```terraform
+data "twingate_group" "foo" {
+  id = "<your group's id>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) The ID of the Group
```

Comment:
Change to:

`The ID of the Group. The ID for the Group must be obtained from the Admin API.`

Reply:
updated description